### PR TITLE
Mandiant Advantage Threat Intelligence: New App - Initial Release

### PR DESCRIPTION
**Pull request by chrishultin**
…SOAR

Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [x] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Initial release of Mandiant Advantage Threat Intelligence integration with Splunk SOAR

## What is the current behavior? (OPTIONAL)
- N/A

## What is the new behavior? (OPTIONAL)
- Adds the ability to query the Mandiant Advantage Threat Intelligence API for information about Indicators, Threat Actors, and Malware Families.

## Other information (OPTIONAL)
- N/A

## Pay close attention to (OPTIONAL)
- N/A

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by pdesai-crest** [comment]
Based on the best practices followed, we have updated:

- Dark theme logo
- Data type of "api_key" configuration parameter from string to password

Also updated minimum phantom version to 5.5.0. Could you please check the working of app on 5.5.0 phantom version?

**Comment by pdesai-crest** [comment]
Hello @chrishultin, could you please check the working of app on the updated phantom version i.e., 5.5.0 version?

**Comment by chrishultin** [comment]
> Hello @chrishultin, could you please check the working of app on the updated phantom version i.e., 5.5.0 version?

I've tested and validated that the app appears to be working on Splunk SOAR 6.0.0.  I did notice and resolve an issue with the `search` function.

**Comment by chrishultin** [comment]
@pdesai-crest 

> Hello @chrishultin, could you please check the working of app on the updated phantom version i.e., 5.5.0 version?

I've tested and validated that the app appears to be working on Splunk SOAR 6.0.0.  I did notice and resolve an issue with the `search` function.

**Review by dhwanis-crest — APPROVED** [review]

**Review by splunk-jessica — APPROVED** [review]

File: .DS_Store
Comment (tonyc-phantom) [inline .DS_Store:1]: Did you mean to commit this file?

**Review by tonyc-phantom — APPROVED** [review]

File: .DS_Store
Comment (pdesai-crest) [reply .DS_Store:1]: I have made the changes and removed the file.

**Review by ishans-crest — APPROVED** [review]

**Pull request merged**